e2e/terraform/README.md

```diff
@@ -102,6 +102,7 @@ $ source ~/.profile
 ```
 
 ## Creating the Config For Cluster
+
 - We will be generating a terraform file(.tf) that will later spawn:
```

Comment:
Who is we? OpenEBS? 
...that will spawn (delete later)

Reply:
I think I will change it to " The script will generate" is that fine?